api/src/main/java/io/minio/MinioClient.java

```diff
@@ -957,6 +957,8 @@ private Request createRequest(Method method, String bucketName, String objectNam
           len = 0;
         }
 
+        // Issue #820. Work-around for okhttp3 default Accept-Encoding gzip
```

Comment:
```suggestion
        // Set the `Accept-encoding` header to `identity` as okhttp will add an 
        // `Accept-Encoding` header for transparent response compression (gzip) unless 
        // the header is already present.
```

Reply:
This comment is too verbose and pain to read. My suggestion is to have just `disable default gzip compression`